src/NeptuneSchema.js

```diff
@@ -271,26 +255,11 @@ async function getNodesProperties() {
 
 
 async function checkEdgeDirectionCardinality(d) {
-    let parameters = `{"from": "${d.from}", "edge": "${d.edge.label}", "to": "${d.to}"}`;
-    let queryFrom = `MATCH(from)-[r]->(to)
-                    WHERE $from in labels(from)
-                        AND type(r) = $edge
-                        AND $to in labels(to)
-                    WITH to, count(from) as rels
-                    WHERE rels > 1
-                    RETURN rels LIMIT 1`;
-    loggerLog(`Checking edge direction cardinality: ${queryFrom}`);
```

Comment:
Is this log dropped intentionally?

Reply:
Not intentional, added back a generic log message.